inductiva/utils/files.py

```diff
@@ -130,7 +130,9 @@ def _unzip(zip_path: pathlib.Path, dest_path: pathlib.Path):
     zip_path.unlink()
 
 
-def download_from_url(url: str, save_dir: Optional[str] = None) -> str:
+def download_from_url(url: str,
+                      save_dir: Optional[str] = None,
+                      unzip: bool = False) -> str:
```

Comment:
this is still a bit surprising for the user, I think... because the method can download any file, but there is an option that is called "unzip", which actually only applies to downloading zip files... What if I pass here "True" and it is not a zip file? I guess it will be ignored...

Reply:
When we pass unzip = True for a zip file, what I would expect from the description is that file would be unzipped to a local folder, with all its contents. But in fact this only applies to zip files with one single file... and there is more: then that file will be renamed (not super clear to what -- the docstring mentions 'local_file_path' but that's unclear...)

Reply:
My take: isn't this method trying to do too many things?
Feels like it should only download a file, that's it. Or if we pass an unzip option, it should not be surprising: just decompress it all, independently of number of files, and do not do any renaming.

Seems like this method is doing all this, because historically we wanted to use it in some snippets of documentation and we want the code there to be simple and compact, but that's not a strong reason enough, to create a method that is super "entangled" with many behaviours inside it... 
Probably we just need a simpler way to store example files of the API, but not implement a method with surprising behaviours. 
What do you think?

Reply:
I see your points! I will look into this and make it simpler and not so entangled.